index.html

```diff
@@ -889,10 +928,10 @@ <h2>Relocating a member of our section away from it</h2>
                 Using a target address instead of a section ensures we deliver the message even if the destination
                 splits or merges.<br />
                 <br />
-                There are 3 states the node may be: State::RelocatingAgeIncrease, State::RelocatingHop, and
-                State::RelocatingBackOnline.<br />
-                Nodes will be selected in this order: State::RelocatingAgeIncrease, State::RelocatingHop and then
-                State::RelocatingBackOnline and tie break by age then name.<br />
+				There are 4 states the node may be in: State::InitialRelocation, State::RelocatingAgeIncrease,
+				State::RelocatingHop, and State::RelocatingBackOnline.<br />
+                Nodes will be selected in this order: State::RelocatingAgeIncrease, State::RelocatingHop,
+				State::InitialRelocation and then State::RelocatingBackOnline and tie break by age then name.<br />
```

Comment:
I think I'd put State::InitialRelocation as the last to take on since the other ones are already older.